src/server/saved_objects/client/lib/search_dsl/sorting_params.test.js

```diff
@@ -119,5 +192,25 @@ describe('searchDsl/getSortParams', () => {
           });
       });
     });
+    describe('sortField is root multi-field with multiple types', () => {
```

Comment:
For these descriptions, maybe say "data type" whenever it's talking about a data type, instead of just "type", which can refer to more things, like mapping types.

Reply:
When using "type" in these descriptions, it's referring to the function parameter which is currently named `type`, so I'm not sure if referring to them as "data type" would provide any more clarity.

Reply:
Fair enough.